src/emulator/functionsEmulatorShared.ts

```diff
@@ -14,6 +14,7 @@ export enum EmulatedTriggerType {
 export interface EmulatedTriggerDefinition {
   entryPoint: string;
   name: string;
+  id?: string;
```

Comment:
Can you add a doc comment here so we know how `id` differs from `name` and when `id` does/does not exist?

Reply:
done